packages/mongodb-rag-ingest/src/pages/updatePages.ts

```diff
@@ -1,36 +1,41 @@
 import { logger, Page, PageStore } from "mongodb-rag-core";
 import { getChangedPages } from "./getChangedPages";
 import { DataSource } from "../sources/DataSource";
-
+import { PromisePool } from "@supercharge/promise-pool";
+import { ConcurrencyOptions } from "../Config";
 /**
   Fetches pages from data sources and stores those that have changed in the data
   store.
  */
 export const updatePages = async ({
   sources,
   pageStore,
+  pageConcurrencyOptions,
 }: {
   sources: DataSource[];
   pageStore: PageStore;
+  pageConcurrencyOptions?: ConcurrencyOptions["pages"]
```

Comment:
same comments as for the `updateEmbeddedContent` about the parameter naming + typescript typing